test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -27,10 +27,10 @@ std::string expected;
 int option_value{5};
 bool flag_value{};
 std::vector<std::string> pos_opt_value{};
-const char * argv0[] = {"./help_add_test"};
-const char * argv1[] = {"./help_add_test", "-h"};
-const char * argv2[] = {"./help_add_test", "-hh"};
-const char * argv3[] = {"./help_add_test", "--version"};
+const char * argv0[] = {"./help_add_test --version-check 0"};
```

Comment:
we said in the message that this parameter takes the same letters as you give interactively, but here you use `0`.